src/pkg/reg/filter/artifact.go

```diff
@@ -102,6 +104,8 @@ func (a *artifactTypeFilter) Filter(artifacts []*model.Artifact) ([]*model.Artif
 // in the filter is the valid one
 type artifactLabelFilter struct {
 	labels []string
+	// "matches", "excludes"
```

Comment:
a little confusing here.

Are we migrating the filters in the replication part to the filters defined in the lib package? Why do we define new filters here?

Reply:
replication is just using the filters defined in this package, I just add a new field `decoration`